castoredc_api/tests/test_api_endpoints/test_country_endpoints.py

```diff
@@ -86,22 +86,22 @@ def test_single_country_failure_too_large(self, client):
         """Tests if retrieving a non-existent (edge case: upper range) country ID raises an error."""
         with pytest.raises(HTTPStatusError) as e:
             client.single_country(252)
-        assert "404 Client Error: Not Found for url" in str(e.value)
+        assert "Client error '404 Not Found" in str(e.value)
```

Comment:
This test, and the ones below and in other testcases, would probably be less brittle if you check the status code as opposed to the string value. Off the top of my head, I believe the attribute is `response.status_code`:

```suggestion
        assert e.response.status_code == 404
```